### ISSUE TITLE
Line-by-line tilt correction

### ISSUE THREAD

**Issue by pastewka** [ContactEngineering/topobank#74]
We should add the option to do a line-by-line tilt correction for uploaded surfaces.

**Comment by mcrot** [comment]
All other detrend modes have been implemented on level of package `SurfaceTopography`.  It looks like
that there is no line-by-line tile correction yet. I guess it should be implemented in `SurfaceTopography`, right?

**Comment by pastewka** [comment]
Yes... please ignore this for now, we should move it to the next release.
